.travis.yml

```diff
@@ -1,16 +1,36 @@
 # A Travis CI configuration file.
 
+sudo: required
+
 language: go
 
 go:
   - 1.7
 
+services:
+  - docker
+
+git:
+  depth: 3
+
 install:
+  - export DEPLOY_BUILD_READY=false
   - go get -u github.com/golang/lint/golint
+  - go get -u github.com/stretchr/testify
 
 script:
   - make lint
   - make build
+  - source ./install_dependencies.sh
```

Comment:
I am thinking of it as well about how to check the openwhisk is ready. This can come later. If openwhisk is not ready, integration tests will fail anyway.